player/migrations/0001_initial.py

```diff
@@ -1,22 +1,32 @@
+<<<<<<< HEAD
```

Comment:
You still have conflict markers that were missed during conflicts resolution.

Reply:
I'd suggest redoing the rebase and accepting all changes coming from upstream/main.

Reply:
Ping me tomorrow and we'll try to figure this out.